api/src/test/java/org/openmrs/module/fhir2/providers/r4/ConditionFhirResourceProviderTest.java

```diff
@@ -143,6 +146,35 @@ public void shouldCreateNewCondition() {
 		assertThat(result.getResource(), equalTo(condition));
 	}
```

Comment:
Please add a test for missing id-
`updateCondition_shouldThrowInvalidRequestExceptionForMissingId`

Reply:
finished thanks